types/last-element/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for last-element 1.0
+// Project: https://github.com/Prosen-Ghosh/last-element#readme
+// Definitions by: Rajas Paranjpe <https://github.com/ChocolateLoverRaj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function last <T>(arr: readonly T[]): T;
```

Comment:
```suggestion
export as namespace last;

declare function last <T>(arr: readonly T[]): T;
```
It also works in the browser via script inclusion

Reply:
@peterblazejewicz 
So would doing it this way make `last` a global function?